packages/api/src/external/carequality/models/cq-directory.ts

```diff
@@ -10,15 +10,18 @@ export class CQDirectoryEntryModel
   static NAME = "cq_directory_entry";
   declare id: string; // Organization's OID
   declare name?: string;
-  declare urlXCPD: string;
+  declare urlXCPD?: string | null;
```

Comment:
I'd think `null` is needed here since this is the model and this column is nullable. But other columns have optional/undefined, so maybe we can also avoid `null` here?